src/statick_tooling/plugins/tool/dockerfile_lint_tool_plugin.py

```diff
@@ -139,5 +139,16 @@ def parse_output(self, total_output: List[str]) -> List[Issue]:
 
             except ValueError as ex:
                 logging.warning("ValueError: %s", ex)
```

Comment:
Do you still want this logging output? Or would it be enough to create the issue? I lean towards removing the logging.

Reply:
I also think the log statements are duplicative. I've removed them.